### PR TITLE
Speedup cleanup

### PR DESCRIPTION
**Pull request by TheRatG**
- [ ] Bug fix #…?
- [ ] New feature?
- [ ] BC breaks?
- [ ] Deprecations?
- [ ] Tests added?
- [x] Changelog updated?

Speedup cleanup. Some update after we faced `rm -rf ` 3-4 minutes long.

- https://unix.stackexchange.com/questions/37329/efficiently-delete-large-directory-containing-thousands-of-files
-  https://www.kinamo.be/en/support/faq/efficiently-remove-a-zillion-files-on-linux-servers

### CODE REVIEW

**Comment by antonmedv** [comment]
Looks like some checks are failing.

**Comment by TheRatG** [comment]
> Looks like some checks are failing.

fixed

File: recipe/deploy/cleanup.php [hunk 1]
```diff
@@ -18,6 +18,7 @@
     }
 
     foreach ($releases as $release) {
+        run("ionice -c 3 nice -19 find {{deploy_path}}/releases/$release/ -delete", $runOpts);
```
Comment (staabm) [inline recipe/deploy/cleanup.php:21]: I think this line deserves a comment, what it is expected todo.
never heard of this command.

Comment (antonmedv) [reply recipe/deploy/cleanup.php:21]: Will it work on all *nix systems? Solaris? Bsd?

Comment (antonmedv) [reply recipe/deploy/cleanup.php:21]: Not sure if this is a solution for general case. Maybe your setup is too specific.

Comment (TheRatG) [reply recipe/deploy/cleanup.php:21]: Yep, I should delete ionice, Bsd doesn't support.

File: recipe/deploy/cleanup.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
     }
 
     foreach ($releases as $release) {
-        run("ionice -c 3 nice -19 find {{deploy_path}}/releases/$release/ -delete", $runOpts);
+        run("find {{deploy_path}}/releases/$release/ -delete", $runOpts);
```
Comment (antonmedv) [inline recipe/deploy/cleanup.php:21]: So what is performance gain? rm rf vs find?

Comment (TheRatG) [reply recipe/deploy/cleanup.php:21]: I found the problem with my deploy, it was mount option **discard**. So I will close request.

**Comment by TheRatG** [comment]
Thanks for support.

**Pull request closed without merging**